napari/_qt/widgets/_tests/test_qt_large_int_spinbox.py

```diff
@@ -51,7 +51,7 @@ def test_event_emits_int(qtbot):
     dtype = np.dtype(np.int32)
 
     def fn(value):
-        assert isinstance(value, dtype.type)
+        assert np.issubdtype(type(value), np.integer)
```

Comment:
```suggestion
        assert np.issubdtype(type(value), int)
```
(I know it's the same thing ... but the `valueChanged` signal is declared as `builtins.int` ... so might as well use that here too)